reference/docs-conceptual/dev-cross-plat/vscode/using-vscode-for-debugging-compiled-cmdlets.md

```diff
@@ -0,0 +1,173 @@
+it. If the output contains the text `Project file does not exist`, then you should edit the
+`tasks.json` file to include the explicit path to the C# project, which can be expressed as
+`"${workspaceFolder}/myModule"` (where `myModule` is the name of the project folder, if applicable).
+
+This needs to go after the `build` entry in the `args` list, as follows:
+
+```json
+    {
+        "label": "build",
+        "command": "dotnet",
+        "type": "shell",
+        "args": [
+            "build",
+            "${workspaceFolder}/myModule",
+            // Ask dotnet build to generate full paths for file names.
+            "/property:GenerateFullPaths=true",
+            // Do not generate summary otherwise it leads to duplicate errors in Problems panel
+            "/consoleloggerparameters:NoSummary",
+        ],
+        "group": "build",
+        "presentation": {
+            "reveal": "silent"
+        },
+        "problemMatcher": "$msCompile"
+    }
+```
+
+## Setting up the debugger
+
+To debug the PowerShell cmdlet, you will need to set up a custom launch configuration. This
+configuration will:
+
+- Build your source code
+
+- Start PowerShell with your module loaded
+
+- Leave PowerShell open in the terminal pane, so that you can invoke your cmdlet
+
+- When you invoke your cmdlet, the debugger will stop at any breakpoints in your source code
+
+The steps are:
+
+1. Install the [C# for Visual Studio Code][] extension
+
+2. In the Debug pane, add a debug configuration
+
+3. In the `Select environment` dialog, choose `.NET Core`
+
+4. This will open `launch.json`. With your cursor inside the `configurations` array, you will see the
+`configuration` picker. If you do not see this, click on `Add Configuration`.
+
+5. Choose `Launch .NET Core Console App`:
+
+   ![Launch .NET Core Console App](media/using-vscode-for-debugging-compiled-cmdlets/add-configuration-dialog.png)
+
+   This will create a default debug configuration.
+
+6. Edit the `name`, `program`, `args` and `console` fields as follows:
+
+   ```json
+    {
+        "name": "PowerShell cmdlets",
+        "type": "coreclr",
+        "request": "launch",
+        "preLaunchTask": "build",
+        "program": "pwsh",
+        "args": [
+            "-NoExit",
+            "-NoProfile",
+            "-Command",
+            "Import-Module ${workspaceFolder}/myModule/bin/Debug/netstandard2.0/myModule.dll",
+        ],
+        "cwd": "${workspaceFolder}",
+        "stopAtEntry": false,
+        "console": "integratedTerminal"
+    }
+   ```
+
+### Explanation of the edits
+
+- It is necessary to launch `pwsh` so that the cmdlet being debugged can be run
+
+- `powershell.exe` will not work with these steps
+
+- The `-NoExit` argument prevents the PowerShell session from exiting as soon as the module is
+imported
+
+- The path in the `Import-Module` argument is the default build output path if you have followed the
```

Comment:
I don't know if you want to call out that the user can also import their module manifest file (`psd1`) and that should work as well.